AirportTicketNookingTest/RoleFactoryShould.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Airport_Ticket_Booking;
+using FluentAssertions;
+using Xunit;
+using static Airport_Ticket_Booking.Common;
+
+namespace AirportTicketNookingTest
+{
+    public class RoleFactoryShould
+    {
+        [Fact]
+        public void returnManagerObjectWhenManagerNeeded()
+        {
+            var manager = RoleFactory.CreateRole((int )RoleEnum.Manager);
+            Assert.NotNull(manager);
+            manager.Should().BeOfType<Manager>();
```

Comment:
is it safe to remove `Assert.NotNull(manager);`? what would happen if we remove it and the object was null? would `manager.Should().BeOfType<Manager>();` handle that case?